backend/LexBoxApi/GraphQL/LexAuthUserCanDownloadProjectsExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using LexCore.Auth;
+using LexCore.Entities;
+
+namespace LexBoxApi.GraphQL;
+
+public static class LexAuthUserCanDownloadProjectsExtensions
+{
+    public static bool CanDownloadProjectsWithoutMembership(this LexAuthUser user)
```

Comment:
let's just make this a method of `LexAuthUser`

Reply:
Done in commit 04976873d.